src/Sarif.Multitool/Rules/RuleResources.resx

```diff
@@ -204,4 +204,16 @@
   <data name="SARIF1007_RegionPropertiesMustBeConsistent_FullDescription_Text" xml:space="preserve">
     <value>Placeholder_SARIF1007_RegionPropertiesMustBeConsistent_FullDescription_Text</value>
   </data>
+  <data name="SARIF2005_ProvideHelpfulToolInformation_FullDescription_Text" xml:space="preserve">
+    <value>Placeholder</value>
+  </data>
+  <data name="SARIF2005_ProvideHelpfulToolInformation_Warning_ProvideConciseToolName_Text" xml:space="preserve">
+    <value>{0}: Placeholder {1} {2}</value>
```

Comment:
>{1} {2} [](start = 28, length = 7)

Put single quotes around the replacement sequences (except for the `{0}`. So: `{0}: Placeholder '{1}' '{2}'`

Reply:
The reason is that the replacement values might be strings with spaces in them, or even empty strings, so we want to see clearly where each replacement value is. In the real messages, I won't put quotes around integer values.

---
In reply to: [444574154](https://github.com/microsoft/sarif-sdk/pull/1926#discussion_r444574154) [](ancestors = 444574154)